src/one_dragon/base/push/channel/work_wexin.py

```
@@ -0,0 +1,199 @@
+            error_messages: 错误消息列表
+
+        Returns:
+            bool: 是否发送成功
+        """
+        try:
+            # 企业微信机器人图片最大支持2MB
+            TARGET_SIZE = 2 * 1024 * 1024
+            img_bytes = self.image_to_bytes(image, max_bytes=TARGET_SIZE).getvalue()
+            img_size = len(img_bytes)
+            img_base64 = base64.b64encode(img_bytes).decode('utf-8')
+            img_md5 = hashlib.md5(img_bytes).hexdigest()
```

Ruff (0.14.0)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)